cdap-ui/app/cdap/components/CaskWizards/UploadDataUsecase/index.js

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright © 2017 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import React, {PropTypes} from 'react';
+import UploadDataWizard from 'components/CaskWizards/UploadData';
+
+export default function UploadDataUsecaseWizard({input, onClose, isOpen}) {
+  return (
+    <UploadDataWizard
+      isOpen={isOpen}
+      input={input}
+      onClose={onClose}
+      isUsecase={true}
```

Comment:
Ideally the successInfo should be here. In reference to this comment: https://github.com/caskdata/cdap/pull/8005/files#r102091344